src/Servant/Swagger/Internal/Orphans.hs

```diff
@@ -1,16 +1,27 @@
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -fno-warn-orphans       #-}
+{-# LANGUAGE CPP                        #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE StandaloneDeriving         #-}
 module Servant.Swagger.Internal.Orphans where
 
 import           Data.Proxy
                  (Proxy (..))
 import           Data.Swagger
 import           Servant.Types.SourceT
                  (SourceT)
+#if __GLASGOW_HASKELL__ >= 881
+import           Servant.API (WithStatus(..))
+#endif
 
 -- | Pretend that 'SourceT m a' is '[a]'.
 --
 -- @since 1.1.7
 --
 instance ToSchema a => ToSchema (SourceT m a) where
     declareNamedSchema _ = declareNamedSchema (Proxy :: Proxy [a])
+
+-- @since <TODO>
+#if __GLASGOW_HASKELL__ >= 881
```

Comment:
```suggestion
#if __GLASGOW_HASKELL__ >= 881
-- @since <TODO>
```

Not sure if this breaks anything, but it seems more honest to me.  (Please ignore if you're no more confident than me.)

Reply:
Hh, yes, missed that. Done